tools/pre-commit.sh

```diff
@@ -7,44 +7,13 @@
 # To do this, execute, from repo root
 # ln -s $(pwd)/tools/pre-commit.sh $(pwd)/.git/hooks/pre-commit
 
-rm -f /tmp/check_apache_header
-rm -f /tmp/check_java_package
-RESULT=0
+bazel query 'kind(java_*, //...) - kind(java_proto_library, //...) - //third_party/... - attr("tags", "checkstyle_ignore", //...)' | sort >/tmp/all_java_targets
```

Comment:
Can you please document this script?

Reply:
@oferb fixed in https://github.com/google/startup-os/pull/486/commits/70e8e9de1d93cdafea77bd4488c6318c5c648a36